packages/react-button/src/components/Button/Button.stories.tsx

```diff
@@ -2,64 +2,74 @@ import * as React from 'react';
 import { Button } from './Button';
 import { ButtonProps } from './Button.types';
 import * as classes from './Button.stories.scss';
-import { Stack, Text } from 'office-ui-fabric-react';
+
+/**
+ * Temporary Stack until there's one in its own package.
+ */
+const Stack = (props: React.PropsWithChildren<{ horizontal?: boolean }>) => {
+  const { horizontal, ...rest } = props;
+
+  return <div {...rest} className={horizontal ? classes.hStack : classes.vStack} />;
+};
+
+const Text = (props: React.PropsWithChildren<{}>) => <h2 {...props} className={classes.text} />;
```

Comment:
nit: You should either indicate in the comment above `Temporary Stack and Text components ...` or add a similar comment here for `Text`.